wrapping/itkSplitComponentsImageFilter.wrap

```diff
@@ -1,9 +1,36 @@
 itk_wrap_class("itk::SplitComponentsImageFilter" POINTER)
+  # We can only wrap types that have been wrapped for
+  # ImageToImageFilter (parent class).
+  UNIQUE(types "${WRAP_ITK_REAL};UC")
   foreach(d ${ITK_WRAP_IMAGE_DIMS})
-    foreach(t ${WRAP_ITK_REAL})
+    foreach(t ${types})
+      if(DEFINED ITKT_IV${t}${d}${d})
       itk_wrap_template(
         "${ITKM_IV${t}${d}${d}}${ITKM_I${t}${d}}"
         "${ITKT_IV${t}${d}${d}}, ${ITKT_I${t}${d}}")
+      endif()
+    endforeach()
+  endforeach()
+  foreach(d ${ITK_WRAP_IMAGE_DIMS})
+    UNIQUE(types "${WRAP_ITK_SCALAR};UC;D;US;UI;UL;SC;SS;SI;SL;F")
+      # Define filter for all image types that are wrapped.
```

Comment:
Seems like there should be a variable that provides what you need here, but I assume you have already considered that :-)

Reply:
As far as I know, there is no variable that does that, but yes, we should add one because the same code is used in the NumPy bridge, so having it in one place would help maintainance. I will add an issue in ITK.

Reply:
Actually, I have already added a variable to get the list of the image scalar types that are wrapped, but it has been added after ITKv5.01rc1 which is the package that is downloaded for CI builds. When a new ITK package is created, I will update the code here.